src/databricks/labs/ucx/hive_metastore/workflows.py

```diff
@@ -112,7 +133,9 @@ def migrate_views(self, ctx: RuntimeContext):
         """This workflow task migrates views from the Hive Metastore to the Unity Catalog using create view sql statement.
         It is dependent on the migration of the tables.
         """
-        ctx.tables_migrator.migrate_tables(what=What.VIEW, acl_strategy=[AclMigrationWhat.LEGACY_TACL])
+        ctx.tables_migrator.migrate_tables(
+            what=What.VIEW, acl_strategy=[AclMigrationWhat.LEGACY_TACL, AclMigrationWhat.PRINCIPAL]
```

Comment:
nit - add a trailing comma so `make fmt` would align it nicely
```suggestion
            what=What.VIEW, acl_strategy=[AclMigrationWhat.LEGACY_TACL, AclMigrationWhat.PRINCIPAL],
```

Reply:
done